src/extensions/surveys.tsx

```diff
@@ -424,6 +433,16 @@ export class SurveyManager {
             : true
 
         if (!internalTargetingFlagCheck) {
+            // check if survey is in progress
+            const isSurveyInProgress = getInProgressSurveyState(survey)
+            if (isSurveyInProgress) {
+                renderReason.disabledReason = `survey is in progress`
```

Comment:
this is odd since the property is called `disabledReason`, should this be set to null instead

Reply:
This is not resolved

Reply:
I resolved because this change doesn't exist anymore, as we don't change that method and the logic is now on the survey eligibility instead